wicket-core/src/main/java/org/apache/wicket/Session.java

```diff
@@ -126,6 +127,11 @@
 	{
 		private static final long serialVersionUID = 1L;
 	};
+	/** records if pages have been unlocked for the current request */
```

Comment:
this javadoc is wrong. it seems to be a copy from somewhere else

Reply:
My bad, will fix ASAP :)

Reply:
should be fixed